### PR TITLE
Jormun: make it possible to use codes from another source

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

File: source/jormungandr/jormungandr/realtime_schedule/realtime_proxy_manager.py [hunk 1]
```diff
@@ -71,7 +72,7 @@ def __init__(self, proxies_configuration):
                 continue
 
             try:
-                rt_proxy = attr(id=proxy_id, **args)  # all services must have an ID
+                rt_proxy = attr(id=proxy_id, code_source=code_source, **args)  # all services must have an ID
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/realtime_schedule/realtime_proxy_manager.py]: I don't think `code_source` is very explicit. maybe `object_realtime_id_tag` ? or `object_id_tag` ?

File: source/jormungandr/jormungandr/schedule.py [hunk 2]
```diff
@@ -173,18 +173,18 @@ def __hash__(self):
         return hash(self.__key())
 
     @staticmethod
-    def _get_code(obj, rt_proxy_id):
-        return next((c.value for c in obj.codes if c.type == rt_proxy_id), None)
+    def _get_code(obj, source):
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/schedule.py]: `realtime_system_id` ?

File: source/jormungandr/jormungandr/schedule.py [hunk 3]
```diff
@@ -173,18 +173,18 @@ def __hash__(self):
...
 
-    def fetch_line_id(self, rt_proxy_id):
-        return self._get_code(self.pb_route.line, rt_proxy_id)
+    def fetch_line_id(self, realtime_system_id):
+        return self._get_code(self.pb_route.line, realtime_system_id)
 
-    def fetch_route_id(self, rt_proxy_id):
-        return self._get_code(self.pb_route, rt_proxy_id)
+    def fetch_route_id(self, realtime_system_id):
+        return self._get_code(self.pb_route, realtime_system_id)
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/schedule.py]: You missrename these vars.

File: source/jormungandr/tests/proxy_realtime_tests.py [hunk 4]
```diff
@@ -197,3 +201,26 @@ def test_departures_realtime_informations(self):
...
+    def test_departure_with_another_source(self):
+        query = self.query_template.format(sp='C:S1', dt='20160102T1000', data_freshness='')
+        response = self.query_region(query)
+        stop_schedules = response['stop_schedules'][0]['date_times']
+        next_passage_dts = [dt["date_time"] for dt in stop_schedules]
+        print(next_passage_dts)
+        assert ['20160102T104242', '20160102T114242'] == next_passage_dts
+
+        for dt in stop_schedules:
+            assert dt['data_freshness'] == 'realtime'
```
Comment (TeXitoi) [inline source/jormungandr/tests/proxy_realtime_tests.py]: lack final endline.

Comment (xlqian) [inline source/jormungandr/tests/proxy_realtime_tests.py]: done

**Comment by TeXitoi** [comment]
OK for me.

**Comment by xlqian** [comment]
Jenkins, test this please!

**Pull request merged**